.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Configure CI Git User
+        run: |
+          git config --global user.name '@tteltrab'
+          git config --global user.email 'tteltrab@users.noreply.github.com'
+          git remote set-url origin https://$GITHUB_ACTOR:$GITHUB_PAT@github.com/IBM/slack-wrench.git
+        env:
+          GITHUB_PAT: ${{ secrets.GITHUB_PAT }}
```

Comment:
Does NPM rely on this? curious that only this job needs a configured git user

Reply:
I imagine this would need a configured git user if updating tags during the publish. That might have been what that PAT was for below, but hard to say without publishing.

I feel not very strongly (1/10) that we go ahead and publish this so that we can get a 1.0 out and then test that the publish works without after, since it will take a bit of work to remove and validate (including bumping version back on all packages so that I don't accidentally publish 1.0 while testing).